src/wolfgame.py

```diff
@@ -3167,15 +3155,25 @@ def reaper(cli, gameid):
     var.IDLE_WARNED_PM = set()
     chan = botconfig.CHANNEL
 
-    last_day_id = var.DAY_COUNT
+    last_day_id = getattr(var, 'DAY_COUNT', None)
```

Comment:
Use double quotes for strings, also prefer to just set var.DAY_COUNT to 0 in reset() rather than using getattr(). The double quotes for strings applies for every other instance you used single quotes as well.